core/src/test/java/nl/nn/adapterframework/jta/btm/BtmJtaTransactionManagerTest.java

```diff
@@ -39,13 +40,16 @@
 	@BeforeClass
 	public static void ensureBTMisNotActive() {
 		if(TransactionManagerServices.isTransactionManagerRunning()) {
+			TransactionManagerType.BTM.closeConfigurationContext();
 			TransactionManagerServices.getTransactionManager().shutdown();
 		}
 	}
 
 	@AfterClass
 	public static void reinstatePreviousTM() {
-		BTMXADataSourceFactory.createBtmTransactionManager();
+		if(TransactionManagerServices.isTransactionManagerRunning()) {
+			fail("TransactionManager still running");
+		}
```

Comment:
Hier is 'BTMXADataSourceFactory.createBtmTransactionManager()' verdwenen. Klopt dat?